roles/confluent.control_center/tasks/main.yml

```diff
@@ -147,6 +148,7 @@
     group: "{{control_center_group}}"
   notify: restart control center
   when: not control_center_secrets_protection_enabled|bool
+  no_log: "{{mask_secrets|bool}}"
```

Comment:
i dont understand how this task prints anything. can you share a sample

Reply:
Tasks to create config in each component prints the entire configuration. Which has passwords as well. I have attached the complete log file before and after in the JIRA description. It has information like 
```
+confluent.controlcenter.streams.ssl.truststore.location=/var/ssl/private/control_center.truststore.jks
+confluent.controlcenter.streams.ssl.truststore.password=confluenttruststorepass

```

Reply:
oh so its when you use `--diff` that thing is the problem then?

Reply:
Can you put that in the description

Reply:
If diff is the only concern here, we could consider diff:no as an alternative. No logging is awful from an ops perspective. 😎

Reply:
If we don't use diff flag, I don't see any information getting printed on stdout. Do you suggest to close this PR?

Reply:
I suggest replacing `no_log: "{{mask_secrets|bool}}"` with `diff: false`.

Reply:
Thanks @erikgb and @domenicbove.
Can you please have a look again on the PR. The idea is to avoid spitting out sensitive information on console even with `--diff` flag while running the playbook. 
We  have utilised `mask_secret` variable to control the diff flag for sensitive information. 
There are few task which  needs no_log to suppress the information as those are not actually getting printed by the diff flag.